src/coreclr/jit/jitconfigvalues.h

```diff
@@ -279,6 +279,38 @@ CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 0) // Controls the Alt
 #else                                                        // !defined(TARGET_ARM64) && !defined(TARGET_X86)
 CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 1) // Controls the AltJit behavior of NYI stuff
 #endif                                                       // defined(TARGET_ARM64) || defined(TARGET_X86)
+///
+/// JIT Hardware Intrinsics
+///
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
+CONFIG_INTEGER(EnableSSE3_4, W("EnableSSE3_4"), 1) // Enable SSE3, SSSE3, SSE 4.1 and 4.2 instruction set as default
+#endif
+
+#if defined(TARGET_AMD64) || defined(TARGET_X86)
+// Enable AVX instruction set for wide operations as default. When both AVX and SSE3_4 are set, we will use the most
+// capable instruction set available which will prefer AVX over SSE3/4.
+CONFIG_INTEGER(EnableHWIntrinsic, W("EnableHWIntrinsic"), 1) // Enable Base
+CONFIG_INTEGER(EnableSSE, W("EnableSSE"), 1)                 // Enable SSE
+CONFIG_INTEGER(EnableSSE2, W("EnableSSE2"), 1)               // Enable SSE2
+CONFIG_INTEGER(EnableSSE3, W("EnableSSE3"), 1)               // Enable SSE3
+CONFIG_INTEGER(EnableSSSE3, W("EnableSSSE3"), 1)             // Enable SSSE3
+CONFIG_INTEGER(EnableSSE41, W("EnableSSE41"), 1)             // Enable SSE41
+CONFIG_INTEGER(EnableSSE42, W("EnableSSE42"), 1)             // Enable SSE42
+CONFIG_INTEGER(EnableAVX, W("EnableAVX"), 1)                 // Enable AVX
+CONFIG_INTEGER(EnableAVX2, W("EnableAVX2"), 1)               // Enable AVX2
+CONFIG_INTEGER(EnableAVXVNNI, W("EnableAVXVNNI"), 1)         // Enable AVXVNNI
+CONFIG_INTEGER(EnableFMA, W("EnableFMA"), 1)                 // Enable FMA
+CONFIG_INTEGER(EnableAES, W("EnableAES"), 1)                 // Enable AES
+CONFIG_INTEGER(EnableBMI1, W("EnableBMI1"), 1)               // Enable BMI1
+CONFIG_INTEGER(EnableBMI2, W("EnableBMI2"), 1)               // Enable BMI2
+CONFIG_INTEGER(EnableSHA, W("EnableSHA"), 1)                 // Enable SHA
+CONFIG_INTEGER(EnableLZCNT, W("EnableLZCNT"), 1)             // Enable AES
+CONFIG_INTEGER(EnablePCLMULQDQ, W("EnablePCLMULQDQ"), 1)     // Enable PCLMULQDQ
+CONFIG_INTEGER(EnablePOPCNT, W("EnablePOPCNT"), 1)           // Enable POPCNT
+#else                                                        // !defined(TARGET_AMD64) && !defined(TARGET_X86)
+// Enable AVX instruction set for wide operations as default
+CONFIG_INTEGER(EnableAVX, W("EnableAVX"), 0)
+#endif                                                       // !defined(TARGET_AMD64) && !defined(TARGET_X86)
```

Comment:
This should be removed. The logic was all moved to src/coreclr/inc/clrconfigvalues.h